### PR TITLE
Update lists.md

### PR DESCRIPTION
**Pull request by sav-norem**
update lists data type page to include short codes for tabbed examples. updated some text to reflect changes.

### CODE REVIEW

File: docs/data-types/lists.md [hunk 1]
```diff
@@ -35,60 +35,60 @@ See the [complete series of list commands](https://redis.io/commands/?group=list
...
-```
+> LMOVE bikes:repairs bikes:finished LEFT LEFT
+"bike:2"
+> LRANGE bikes:repairs 0 -1
+1) "bike:1"
+> LRANGE bikes:finished 0 -1
+1) "bike:2"
+{{< /clients-example >}}
+
+* To limit the length of a list you can call `LTRIM`:
```
Comment (dwdougherty) [inline docs/data-types/lists.md:81]: I don't know why, but this example doesn't display in my local build. You may have to reach out to Igor to solve this.

File: docs/data-types/lists.md [hunk 2]
```diff
@@ -223,26 +223,38 @@ The `LTRIM` command is similar to `LRANGE`, but **instead of displaying the
...
+    1) "bike:1"
+    2) "bike:2"
+    3) "bike:3"
+{{< /clients-example >}}
 
 The above `LTRIM` command tells Redis to keep just list elements from index
 0 to 2, everything else will be discarded. This allows for a very simple but
 useful pattern: doing a List push operation + a List trim operation together
-in order to add a new element and discard elements exceeding a limit:
+in order to add a new element and discard elements exceeding a limit. Using
```
Comment (dwdougherty) [inline docs/data-types/lists.md:243]:
```suggestion
to add a new element and discard elements exceeding a limit. Using 
```

File: docs/data-types/lists.md [hunk 3]
```diff
@@ -223,26 +223,38 @@ The `LTRIM` command is similar to `LRANGE`, but **instead of displaying the
...
+    2) "bike:2"
+    3) "bike:3"
+{{< /clients-example >}}
 
 The above `LTRIM` command tells Redis to keep just list elements from index
 0 to 2, everything else will be discarded. This allows for a very simple but
 useful pattern: doing a List push operation + a List trim operation together
-in order to add a new element and discard elements exceeding a limit:
+in order to add a new element and discard elements exceeding a limit. Using 
+`LTRIM` with negative indices can then be used to keep only the 3 most recently added:
```
Comment (dwdougherty) [inline docs/data-types/lists.md:244]:
```suggestion
`LTRIM` with negative indexes can then be used to keep only the 3 most recently added:
```

File: docs/data-types/lists.md [hunk 4]
```diff
@@ -35,60 +35,60 @@ See the [complete series of list commands](https://redis.io/commands/?group=list
 ## Examples
```
Comment (dwdougherty) [inline docs/data-types/lists.md:35]: Just a general comment, most of the CLI examples do not use leading whitespace in front of the commands. It's not a big deal, but from a stylistic standpoint, all of examples should be done the same way. Here, there's a mixture.

File: docs/data-types/lists.md [hunk 5]
```diff
@@ -35,60 +35,60 @@ See the [complete series of list commands](https://redis.io/commands/?group=list
...
 
 * Treat a list like a queue (first in, first out):
```
Comment (uglide) [inline docs/data-types/lists.md:37]: We can make this example more self-explanatory by splitting it into two steps:
1. Producer:  LPUSH, RPUSH, etc.
2. Consumer: LPOP, RPOP, LRANGE, etc.

File: docs/data-types/lists.md (hunk 1 above)
Comment (sav-norem) [reply docs/data-types/lists.md:81]: @dwdougherty we resolved this one with the duplicate example yea?

Comment (dwdougherty) [reply docs/data-types/lists.md:81]: @sav-norem Yes.

**Review by dwdougherty — APPROVED** [review]
I built this PR in my test environment and the changes look good.

**Review by dmaier-redislabs — APPROVED** [review]

**Pull request merged**